### PR TITLE
C/CPP : Infix, Prefix and Postfix ( Application of Stack )

### LINKED ISSUE #2327: C/CPP : Infix, Prefix and Postfix ( Application of Stack )
## Description 📜

I will create a folder Stack ( in the CPP Domain ) and inside that folder, I will add documentation to explain Application of Stack - Evaluation of Arithmetic Expressions ( Introduction to Infix, Prefix, and Postfix Expressions with code ).
<hr>

## Domain of Contribution 📊

- [x] C/CPP
- [x] DSA

<hr>

### PR DESCRIPTION
**Pull request by PranaySanjule**
<hr>

## Description 📜

Under the C/CPP domain, In the directory "Data Structures/Stack", I made one more folder "Infix, Prefix & Postfix (Application of Stack) " inside that folder I've added 4 files, first is the documentation to explain the infix, prefix, and postfix expressions with examples and remaining 3 files containing C++ code to evaluate infix, prefix, and postfix expression. Sorry for the commits, I didn't know commits are marked. And as a beginner, I didn't know about this. I hope my files are correct.
Fixes #2327

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] C/CPP

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

<hr>

### CODE REVIEW

File: C_CPP/Data Structures/Stack/Infix, Prefix & Postfix (Application of Stack)/infixEvaluation.cpp [hunk 1]
```diff
@@ -0,0 +1,137 @@
+//Program to evaluate infix expression
+#include <bits/stdc++.h>
```
Comment (deeqakkk) [inline C_CPP/Data Structures/Stack/Infix, Prefix & Postfix (Application of Stack)/infixEvaluation.cpp:2]: Try adding more suitable comments to make it easily understandable.

File: C_CPP/Data Structures/Stack/Infix, Prefix & Postfix (Application of Stack)/postfixEvaluation.cpp [hunk 2]
```diff
@@ -0,0 +1,65 @@
+// Program to evaluate postfix expression.
```
Comment (deeqakkk) [inline C_CPP/Data Structures/Stack/Infix, Prefix & Postfix (Application of Stack)/postfixEvaluation.cpp:1]: same for here Pranay, try adding more relevant comments.

**Review by deeqakkk — CHANGES_REQUESTED** [review]

File: C_CPP/Data Structures/Stack/Infix, Prefix & Postfix (Application of Stack)/infixEvaluation.cpp (hunk 1 above)
Comment (PranaySanjule) [reply C_CPP/Data Structures/Stack/Infix, Prefix & Postfix (Application of Stack)/infixEvaluation.cpp:2]: Ok sir

Comment (PranaySanjule) [reply C_CPP/Data Structures/Stack/Infix, Prefix & Postfix (Application of Stack)/infixEvaluation.cpp:2]: Sir, I made all changes please review it again.

**Comment by PranaySanjule** [comment]
Sir, I made all the changes please review my Pull Request.

**Pull request closed without merging**